doc/reference/reference_sql/sql_user_guide.rst

```diff
@@ -214,7 +214,7 @@ A literal has :ref:`data type = VARBINARY <sql_data_type_varbinary>`
 
 Here are four ways to put non-ASCII characters,such as the Greek letter α alpha,  in string literals: |br|
 First make sure that your shell program is set to accept characters as UTF-8. A simple way to check is |br|
-``SELECT hex('α');``
+``SELECT hex(cast('α' as VARBINARY));``
 If the result is CEB1 -- which is the hexadecimal value for the UTF-8 representation of α -- it is good. |br|
 (1) Simply enclose the character inside ``'...'``, |br|
```

Comment:
@pgulutzan, this looks like a list, should we maybe format it like one? I mean, in ReStructuredText formatting.

Reply:
That might be a good idea, but I think it should be considered for a different pull request. As you might have noticed, there are now some more data types, so after this is merged I will probably need to make a new pull request for "Even more behavior changes in SQL".

Reply:
@patiencedaur should we make a task out of it and put it in backlog for now?

Reply:
@alexandra-mara Yes please. Can you create an issue so we don't forget about it?

Reply:
@patiencedaur: In the pull request https://github.com/tarantool/doc/pull/2523 you will see that I formatted the list with bullets.